internal/snmp/translate.go

```diff
@@ -145,14 +145,19 @@ func GetIndex(oidNum string, mibPrefix string) (col []string, tagOids map[string
 }
 
 //nolint:revive //Too many return variable but necessary
-func SnmpTranslateCall(oid string) (mibName string, oidNum string, oidText string, conversion string, err error) {
+func SnmpTranslateCall(oid string) (mibName string, oidNum string, oidText string, conversion string, module gosmi.SmiModule, node gosmi.SmiNode, err error) {
```

Comment:
if returning the module, then there is no need to also return `mibName`, as You can just do `module.Name`.

Reply:
No longer returning `module` as it is not needed but `node` is. I would rather pass around a `string` than a `gosmi.SmiModule`.